src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgeryTokenGenerator.cs

```diff
@@ -66,7 +66,12 @@ public AntiforgeryToken GenerateCookieToken()
             if (identity != null && identity.IsAuthenticated)
```

Comment:
Also, why does caller pass in `httpContext.User.Identity` when this code turns around and uses `httpContext.User?.Identities`? Another reason to start w/ the `ClaimsPrinciple` as @rynowak suggested [below](https://github.com/aspnet/Antiforgery/pull/52/files#r52238020).